ui/app/templates/components/pki-cert-show.hbs

```diff
@@ -13,18 +13,28 @@
   <MessageError @model={{model}} />
   {{#each model.attrs as |attr|}}
     {{#if (eq attr.type "object")}}
-      <InfoTableRow @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}} @value={{stringify (get model attr.name)}} />
+      <InfoTableRow data-test-table-row 
+        @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}} 
+        @value={{stringify (get model attr.name)}} />
     {{else}}
       {{#if attr.options.masked}}
-        <InfoTableRow @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}} @value={{get model attr.name}}>
+        <InfoTableRow data-test-table-row 
+          @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}} 
+          @value={{get model attr.name}}>
           <MaskedInput
             @value={{get model attr.name}}
             @displayOnly={{true}}
             @allowCopy={{true}}
           />
         </InfoTableRow>
+      {{else if (and (get model attr.name) (or (eq attr.name "issueDate") (eq attr.name "expiryDate")))}}
```

Comment:
Only pass to date-format helper if `get model attr.name` returns a value AND `attr.name` equals `issueDate` or `expiryDate`